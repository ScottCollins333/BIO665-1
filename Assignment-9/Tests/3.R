suppressPackageStartupMessages(suppressWarnings(suppressMessages(source("3.R"))))

test_data = suppressWarnings(suppressMessages(read_tsv("3.tsv")))
print(test_data)