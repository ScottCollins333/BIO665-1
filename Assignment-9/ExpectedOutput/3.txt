[1] TRUE
# A tibble: 6 x 3
  anemoneNumber startleResponse1 startleResponse2
          <dbl>            <dbl>            <dbl>
1             2              248              268
2             4              350              261
3             6              410              467
4             8              201              188
5            10              687              690
6            12              980              571
